regulations/static/regulations/css/less/typography.less

```diff
@@ -42,35 +42,50 @@ h3,
 h4,
 h5,
 h6 {
-    .font-bold;
-    margin-bottom: 0;
+    .sans-font-bold;
+    margin: 0.5em 0;
```

Comment:
Yeah I think I'll just use the h1-h6 rule to only set the margin then the individual styles for h1 through h6 will each use the appropriate mixin instead of setting bold to all of them and override weights.